roboschool/cpp-household/render-simple.h

```diff
@@ -38,7 +38,7 @@ enum {
 	VIEW_NO_CAPTIONS     = 0x2000,
 };
 
-#define CHECK_GL_ERROR { int e = glGetError(); if (e!=GL_NO_ERROR) fprintf(stderr, "%s:%i ERROR: 0x%x\n", __FILE__, __LINE__, e); assert(e == GL_NO_ERROR); }
+#define CHECK_GL_ERROR { int e = glGetError(); if (e!=GL_NO_ERROR && e!=0x506) fprintf(stderr, "%s:%i ERROR: 0x%x\n", __FILE__, __LINE__, e); assert(e == GL_NO_ERROR || e == 0x506); }
```

Comment:
One day I will figure out exactly why GL throws 0x506 on OS X :) For now, we are just ignoring it.

Reply:
This looks like GL_INVALID_FRAMEBUFFER_OPERATION so maybe the framebuffer is missing some necessary attachment or an invalid texture has been attached.  Does it still work on mac despite the error?

Reply:
Seems to work